tests/topotests/route-scale/test_route_scale.py

```diff
@@ -112,9 +112,42 @@ def test_converge_protocols():
     if tgen.routers_have_failure():
         pytest.skip(tgen.errors)
 
+def run_one_setup(r1, s):
+    "Run one ecmp config"
+
+    global expected_installed, expected_removed
```

Comment:
Would you mind to pass this variables via parameter instead of globally?

I see that they are only read, so I think it might be doable here.

Reply:
sure - I left them global since they never change - the 'expected' json is only set up once at the start of the suite.